exporters/otlp/otlpmetric/internal/oconf/envconfig.go

```diff
@@ -96,6 +98,7 @@ func getOptionsFromEnv() []GenericOption {
 		WithEnvCompression("METRICS_COMPRESSION", func(c Compression) { opts = append(opts, WithCompression(c)) }),
 		envconfig.WithDuration("TIMEOUT", func(d time.Duration) { opts = append(opts, WithTimeout(d)) }),
 		envconfig.WithDuration("METRICS_TIMEOUT", func(d time.Duration) { opts = append(opts, WithTimeout(d)) }),
+		WithEnvTemporalitySelector("METRICS_TEMPORALITY_PREFERENCE", func(s metric.TemporalitySelector) { opts = append(opts, WithTemporalitySelector(s)) }),
```

Comment:
The only documentation which tells about `OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE` support is `CHANGELOG.md`. I think that the support of this env var should be documented in GoDoc for `otlpmetricgrpc` and `otlpmetrichttp`. It may be worth introducing an `# Environment variables` section in `doc.go` files.

Probably other env vars should be also documented as well, but if I recall correctly all/most of them are at least mentioned in option functions. I think we can start with documenting only `OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE` and document other env vars in separate PR(s).

Related to: https://github.com/open-telemetry/opentelemetry-go/issues/2046